README.md

```diff
@@ -183,6 +183,16 @@ Anything-but suffix:
 }
 ```
 
+Anything-but-ignore-case list (strings):
+```javascript
+{
+  "detail": {
+    "state": [ { "anything-but": {"ignore-case": [ "Stopped", "OverLoaded" ] } } ]
```

Comment:
I didn't spot this earlier but this is different than the existing `equals-ignore-case`. While I'm on the fence, I think I still prefer using it over creating a new matcher. It's just weird that we'll have this one matcher that works with single value but then with arrays when use in conjunction with anything-but. 

@timbray  @jonessha  any opinions?

Reply:
Until Tim & Shawn object by Monday (Shawn is away this week), I'd favor going `equals-ignore-case` over `ignore-case`. Definitely will check back by monday EOD.